spec/features/staff_can_view_all_defects_spec.rb

```diff
@@ -162,6 +162,74 @@
     end
   end
 
+  scenario 'defects can be filtered by manual escalations' do
+    flagged_defect = DefectPresenter.new(create(:property_defect, flagged: true))
+    unflagged_defect = DefectPresenter.new(create(:property_defect, flagged: false))
+
+    visit dashboard_path
+
+    click_on('View all defects')
+
+    within('.filter-defects') do
+      uncheck_all_filters
+      check 'Manually escalated', name: 'escalations[]'
+      click_on(I18n.t('generic.button.filter'))
+    end
+
+    within '.defects' do
+      expect(page).to have_content(flagged_defect.reference_number)
+      expect(page).not_to have_content(unflagged_defect.reference_number)
+    end
+  end
+
+  scenario 'defects can be filtered by overdue escalations' do
+    travel_to Time.zone.parse('2019-11-26')
+
+    on_time_defect = create(:property_defect, target_completion_date: Date.new(2019, 11, 29), status: :outstanding)
+    overdue_defect = create(:property_defect, target_completion_date: Date.new(2019, 11, 20), status: :outstanding)
+
+    visit dashboard_path
+
+    click_on('View all defects')
+
+    within('.filter-defects') do
+      uncheck_all_filters
+      check 'Overdue', name: 'escalations[]'
+      click_on(I18n.t('generic.button.filter'))
+    end
+
+    within '.defects' do
+      expect(page).to have_content(overdue_defect.reference_number)
+      expect(page).not_to have_content(on_time_defect.reference_number)
+    end
+
+    travel_back
+  end
+
+  scenario 'defects can be filtered by due soon escalations' do
+    travel_to Time.zone.parse('2019-11-26')
```

Comment:
Can probably have `travel_to` as a block here, to avoid having to make sure you have `travel_back` at the end of the spec.